Tribler/Core/TFTP/handler.py

```diff
@@ -454,6 +476,12 @@ def _handle_packet_as_receiver(self, session, packet):
         self._logger.debug(u"%s Got data, #block = %s size = %s", session, packet['block_number'], len(packet['data']))
 
         # check block_number
+        # ignore old ones, they may be retransmissions
+        if packet['block_number'] < session.block_number:
```

Comment:
I asked @whirm, it's UDP so I am not sure if those packets will arrive in the same order...